database/src/main/scala/com/rasterfoundry/granary/PredictionDao.scala

```diff
@@ -75,4 +83,51 @@ object PredictionDao {
       case None      => Left(ModelNotFound)
     }
   }
+
+  def addResults(
+      predictionId: UUID,
+      webhookId: UUID,
+      status: PredictionStatusUpdate
+  ): OptionT[ConnectionIO, Prediction] =
+    for {
+      existingPrediction <- OptionT(getPrediction(predictionId)) flatMap {
```

Comment:
This would be pretty easy to do db side, but this way I don't have to write any more sql strings, which I think is a positive? And since it's a primary key lookup I decided this is fine